tests/bluetooth/tester/src/btp_bas.c

```diff
@@ -0,0 +1,266 @@
+{
+	const struct btp_bas_bls_set_battery_present_cmd *cp =
+		(const struct btp_bas_bls_set_battery_present_cmd *)cmd;
+	enum bt_bas_bls_battery_present present = cp->present;
+
+	bt_bas_bls_set_battery_present(present);
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t btp_bas_bls_set_wired_power_source(const void *cmd, uint16_t cmd_len, void *rsp,
+						  uint16_t *rsp_len)
+{
+	const struct btp_bas_bls_set_wired_power_source_cmd *cp =
+		(const struct btp_bas_bls_set_wired_power_source_cmd *)cmd;
+	enum bt_bas_bls_wired_power_source source = cp->source;
+
+	bt_bas_bls_set_wired_external_power_source(source);
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t btp_bas_bls_set_wireless_power_source(const void *cmd, uint16_t cmd_len, void *rsp,
+						     uint16_t *rsp_len)
+{
+	const struct btp_bas_bls_set_wireless_power_source_cmd *cp =
+		(const struct btp_bas_bls_set_wireless_power_source_cmd *)cmd;
+	enum bt_bas_bls_wireless_power_source source = cp->source;
+
+	bt_bas_bls_set_wireless_external_power_source(source);
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t btp_bas_bls_set_battery_charge_state(const void *cmd, uint16_t cmd_len, void *rsp,
+						    uint16_t *rsp_len)
+{
+	const struct btp_bas_bls_set_battery_charge_state_cmd *cp =
+		(const struct btp_bas_bls_set_battery_charge_state_cmd *)cmd;
+	enum bt_bas_bls_battery_charge_state state = cp->state;
+
+	bt_bas_bls_set_battery_charge_state(state);
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t btp_bas_bls_set_battery_charge_level(const void *cmd, uint16_t cmd_len, void *rsp,
+						    uint16_t *rsp_len)
+{
+	const struct btp_bas_bls_set_battery_charge_level_cmd *cp =
+		(const struct btp_bas_bls_set_battery_charge_level_cmd *)cmd;
+	enum bt_bas_bls_battery_charge_level level = cp->level;
+
+	bt_bas_bls_set_battery_charge_level(level);
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t btp_bas_bls_set_battery_charge_type(const void *cmd, uint16_t cmd_len, void *rsp,
+						   uint16_t *rsp_len)
+{
+	const struct btp_bas_bls_set_battery_charge_type_cmd *cp =
+		(const struct btp_bas_bls_set_battery_charge_type_cmd *)cmd;
+	enum bt_bas_bls_battery_charge_type type = cp->type;
+
+	bt_bas_bls_set_battery_charge_type(type);
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t btp_bas_bls_set_charging_fault_reason(const void *cmd, uint16_t cmd_len, void *rsp,
+						     uint16_t *rsp_len)
+{
+	const struct btp_bas_bls_set_charging_fault_reason_cmd *cp =
+		(const struct btp_bas_bls_set_charging_fault_reason_cmd *)cmd;
+	enum bt_bas_bls_charging_fault_reason reason = cp->reason;
+
+	bt_bas_bls_set_charging_fault_reason(reason);
+
+	return BTP_STATUS_SUCCESS;
+}
+
+#if defined(CONFIG_BT_BAS_BLS_IDENTIFIER_PRESENT)
+static uint8_t btp_bas_bls_set_identifier(const void *cmd, uint16_t cmd_len, void *rsp,
+					  uint16_t *rsp_len)
+{
+	const struct btp_bas_bls_set_identifier_cmd *cp =
+		(const struct btp_bas_bls_set_identifier_cmd *)cmd;
+	uint16_t identifier = cp->identifier;
```

Comment:
should be converted from LE to host order

Reply:
Thanks for all the feedback!
Could you pls point me to an example of another file that has the correct Host order?